bundles/org.openhab.binding.ambientweather/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.openhab.addons.bundles</groupId>
+		<artifactId>org.openhab.addons.reactor.bundles</artifactId>
+		<version>2.5.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.openhab.binding.ambientweather</artifactId>
+
+	<name>openHAB Add-ons :: Bundles :: Ambient Weather Binding</name>
+
+	<dependencies>
+		<!-- https://mvnrepository.com/artifact/io.socket/socket.io-client -->
+		<dependency>
+			<groupId>io.socket</groupId>
+			<artifactId>socket.io-client</artifactId>
+			<version>1.0.0</version>
+			<scope>provided</scope>
+		</dependency>
+        <!-- https://mvnrepository.com/artifact/io.socket/engine.io-client -->
+        <dependency>
+            <groupId>io.socket</groupId>
+            <artifactId>engine.io-client</artifactId>
+            <version>1.0.0</version>
+            <scope>provided</scope>
+        </dependency>
+		<!-- https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp -->
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
```

Comment:
Is there a reason not to work with the standard HttpUtil or just plan Jetty? And any reason not to use gson?

Reply:
I just saw your chat with @davidgraeff would be nice if you could come back to this binding later to switch to the defaults.

Reply:
I'll investigate if I can get Jetty Websockets to work after this gets merged. I think we're on a newer version of Jetty than when I first tried it.